docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
```

Comment:
what do we need gcc for?

Reply:
some python modules require gcc to be built and that's the case for some of the PySyft's deps

Reply:
+1

Reply:
One way to get rid of gcc is to use a docker multi-stage build and copy the created python installation (environment) to an environment without gcc.

Reply:
that's right, but I'm not sure about all the files that I must copy (especially with pytorch) so I took image size against complexity in this case